### PR TITLE
Fix import_cv2 binary file

### PR DESCRIPTION
**Pull request by qboot**
Hello! 👋

I've been recently trying to work with `import_cv2` and I've needed to make some small changes to get it works:
- Missing `.mp3` extension
- Permission 644 -> 755
- *(Formatting blank spaces)*

Thanks for making Speech-To-Text open source 💪

### CODE REVIEW

**Comment by lissyx** [comment]
> * Missing `.mp3` extension

There's no reason this is needed.

> 
> * Permission 644 -> 755

Similar, there's no good reason to justify ?

File: bin/import_cv2.py [hunk 1]
```diff
@@ -53,23 +53,23 @@ def _preprocess_data(audio_dir, tsv_dir):
...
     counter = { 'all': 0, 'too_short': 0, 'too_long': 0 }
     lock = RLock()
     num_samples = len(samples)
     rows = []
-    
+
     def one_sample(sample):
         """ Take a audio file, and optionally convert it to 16kHz WAV """
-        mp3_filename = path.join(audio_dir, sample[0])
+        mp3_filename = path.join(audio_dir, sample[0]) + ".mp3"
```
Comment (lissyx) [inline bin/import_cv2.py]: That's clearly wrong.

Comment (JRMeyer) [reply bin/import_cv2.py]: @lissyx how? I have this locally and it works

Comment (JRMeyer) [reply bin/import_cv2.py]: @lissyx - the released CV doesn't have extensions for some reason

Comment (lissyx) [reply bin/import_cv2.py]: @JRMeyer Beta had extensions, it makes no sense to not have extensions ...

Comment (lissyx) [reply bin/import_cv2.py]: @qboot Can you make it at least with a check, it really really feels bogus that CV suddenly changed the naming convention.

**Comment by lissyx** [comment]
@qboot Please remove the file mode change as well.

**Comment by lissyx** [comment]
> > * Missing `.mp3` extension
> 
> There's no reason this is needed.
> 
> > * Permission 644 -> 755
> 
> Similar, there's no good reason to justify ?

Nevermind, looks like all ther other `import_*.py` are also `+x` ...

File: bin/import_cv2.py (hunk 1 above)
Comment (kremnik) [reply bin/import_cv2.py]: @lissyx Why it is wrong? Current .tsv files in the Common Voice dataset doesn't have filename extensions, just name. So when you try to convert mp3 files to wav files sting 72 `mp3_filename = path.join(audio_dir, sample[0])` doesn't include .mp3 extension, and converting doesn't work.

**Comment by qboot** [comment]
Thanks for your comments.
Yes, `.mp3` extension is not included in the last french CV dataset that I've downloaded yesterday.
And for the `+x` I just use the same as other importers.

Do I have to do something more in this PR ?

File: bin/import_cv2.py (hunk 1 above)
Comment (lissyx) [reply bin/import_cv2.py]: @kremnik It's wrong because that's not what should be in the dataset: all the beta releases were fine with `.mp3`.

**Comment by lissyx** [comment]
Right, so `.mp3` missing is confirmed to be a bug on Common Voice side. Given there are also beta and other previous dataset in the wild with the proper `.mp3` extension, it'd be really a better approach to check if the filename does have a `.mp3` extension and only add it if missing. cc @qboot @kremnik

File: bin/import_cv2.py [hunk 2]
```diff
@@ -53,23 +53,25 @@ def _preprocess_data(audio_dir, tsv_dir):
...
     counter = { 'all': 0, 'too_short': 0, 'too_long': 0 }
     lock = RLock()
     num_samples = len(samples)
     rows = []
-    
+
     def one_sample(sample):
         """ Take a audio file, and optionally convert it to 16kHz WAV """
         mp3_filename = path.join(audio_dir, sample[0])
+        if not mp3_filename.lower().endswith('.mp3'):
```
Comment (lissyx) [inline bin/import_cv2.py]: nit: I think `path.splitext(mp3_filename.lower())[0] == '.mp3'` would be better :)

Comment (qboot) [reply bin/import_cv2.py]: You mean `path.splitext(mp3_filename.lower())[1] == '.mp3'` nop :) ?
Done.

**Comment by lissyx** [comment]
LGTM, thanks!

**Pull request merged**